### PR TITLE
WD-5869 - feat: Display search errors

### PR DESCRIPTION
**Pull request by vladimir-cucu**
## Done

- Display search errors.

## Details

https://warthogs.atlassian.net/browse/WD-5869

### CODE REVIEW

File: src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx [hunk 1]
```diff
@@ -1,101 +1,28 @@
...
 const ResultsBlock = (): JSX.Element | null => {
   const dispatch = useAppDispatch();
   const isCrossModelQueryLoading = useAppSelector(getCrossModelQueryLoading);
   const isCrossModelQueryLoaded = useAppSelector(getCrossModelQueryLoaded);
   const crossModelQueryResults = useAppSelector(getCrossModelQueryResults);
-  const [codeSnippetView, setCodeSnippetView] = useState<CodeSnippetView>(
-    CodeSnippetView.TREE
-  );
+  const crossModelQueryErrors = useAppSelector(getCrossModelQueryErrors);
   const resultsJSON = JSON.stringify(crossModelQueryResults, null, 2);
```
Comment (huwshimi) [inline src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx:26]: I think we can move this stringify into `CodeSnippetBlock` and then we only need to pass the results or errors in.

**Review by huwshimi — APPROVED** [review]
Looks great. A few small suggestions.

File: src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx (hunk 1 above)
Comment (vladimir-cucu) [reply src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx:26]: That makes so much sense! I'll modify it in my following commit.

File: src/pages/AdvancedSearch/ErrorsBlock/_errors-block.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@import "vanilla-framework/scss/vanilla";
+
+.errors-block {
+  .status-icon::before {
+    left: -$sph--small;
+  }
```
[the remark this answers is not in the document]
Comment (vladimir-cucu) [reply src/pages/AdvancedSearch/ErrorsBlock/_errors-block.scss]: Oh, yeah, sorry. 😅

File: package.json [hunk 3]
```diff
@@ -61,6 +61,7 @@
     "formik": "2.4.1",
     "framer-motion": "10.12.16",
     "fuse.js": "6.6.2",
+    "lodash": "4.17.21",
```
[the remark this answers is not in the document]
Comment (vladimir-cucu) [reply package.json]: Makes sense. Changed it!

**Pull request merged**